test/Confluent.Kafka.IntegrationTests/Tests/Consumer_Assignment.cs

```diff
@@ -40,30 +41,23 @@ public void Consumer_Assignment(string bootstrapServers)
             {
                 GroupId = Guid.NewGuid().ToString(),
                 BootstrapServers = bootstrapServers,
-                SessionTimeoutMs = 6000
+                SessionTimeoutMs = 6000,
```

Comment:
With librdkafka v1.0.0 the default session timeout is 10s, so this is probably not needed anymore.